src/go/rpk/pkg/cli/cluster/partitions/list.go

```diff
@@ -1,207 +1,231 @@
+// Copyright 2023 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
 package partitions
 
 import (
+	"context"
+	"errors"
 	"fmt"
+	"net/http"
 	"strconv"
 	"strings"
+	"sync"
 
-	"github.com/docker/go-units"
 	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/adminapi"
 	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
 	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
 	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
 	"github.com/twmb/types"
+	"go.uber.org/zap"
+	"golang.org/x/sync/errgroup"
 )
 
-func newListPartitionMovementsCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+func newListCommand(fs afero.Fs, p *config.Params) *cobra.Command {
 	var (
-		completion       int
-		all              bool
-		human            bool
-		partitions       []string
-		response         []adminapi.ReconfigurationsResponse
-		filteredResponse []adminapi.ReconfigurationsResponse
+		all          bool
+		disabledOnly bool
+		partitions   []int
 	)
 	cmd := &cobra.Command{
-		Use:   "move-status",
-		Short: "Show ongoing partition movements",
-		Long:  helpListMovement,
-		Run: func(cmd *cobra.Command, topics []string) {
-			p, err := p.LoadVirtualProfile(fs)
-			out.MaybeDie(err, "unable to load config: %v", err)
-			out.CheckExitCloudAdmin(p)
+		Use:     "list [TOPICS]",
+		Aliases: []string{"ls", "describe"},
+		Short:   "List partitions in the cluster",
+		Long: `List partitions in the cluster
```

Comment:
Wonder we should explain what the `DISABLED` column means by along with the meaning of `-`.

Reply:
Added a new paragraph for this :+1: